src/vishwasCDec19/assignment3/Q5RevNo.java

```diff
@@ -0,0 +1,24 @@
+package vishwasCDec19.assignment3;
+/*
+ * WAP to reveres user given number
+ */
+import java.util.Scanner;
+public class Q5RevNo 
+{
+	static void reverse(String number) {
+
+		for (int i = number.length() - 1; i >= 0; i--) 
+		{
+			char r = number.charAt(i);
```

Comment:
what if i will give number as number not String?
then how you will implement logic